### PR TITLE
XSUP-27956/ Added EWS PS V3 Description

### PR DESCRIPTION
**Pull request by AradCarmi**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-27956

### CODE REVIEW

**Review by amshamah419 — APPROVED** [review]
Great work!

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3_description.md [hunk 1]
```diff
@@ -26,16 +26,31 @@ To use this integration, you need to connect an application with a certificate.
...
-   3. In the Certificates tab, upload the publickey.cer file.
+    1. In the Cortex XSOAR Playground, download the publickey.cer file
+    2. In the Azure application, in the left menu, click **Certificates & secrets**.
+    3. In the Certificates tab, upload the publickey.cer file.
+
+
+4. Assign Azure AD roles to the application.
+    1. You have two options:
+        1. Assign Azure AD roles to the application.
+        2. Assign custom role groups to the application using service principals.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3_description.md:37]:
```suggestion
        - Assign Azure AD roles to the application.
        - Assign custom role groups to the application using service principals.
```

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3_description.md [hunk 2]
```diff
@@ -26,16 +26,31 @@ To use this integration, you need to connect an application with a certificate.
...
+    2. In Azure AD portal at https://portal.azure.com/, start typing roles and administrators in the Search box at the
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3_description.md:38]:
```suggestion
    2. In the Azure AD portal at https://portal.azure.com/, start typing roles and administrators in the Search box at the
```

File: Packs/MicrosoftExchangeOnline/ReleaseNotes/1_2_26.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### EWS Extension Online Powershell v3
+
+- Updated the documentation.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/ReleaseNotes/1_2_26.md:6]:
```suggestion
Updated the documentation.
```

**Comment by ShirleyDenkberg** [comment]
@amshamah419 @thefrieddan1 Doc review completed.

**Pull request merged**